src/domain/interfaces/email-api.interface.ts

```diff
@@ -3,5 +3,21 @@ import { CreateEmailMessageDto } from '@/domain/email/entities/create-email-mess
 export const IEmailApi = Symbol('IEmailApi');
 
 export interface IEmailApi {
+  /**
+   * Creates and sends a new email message.
+   *
+   * @param createEmailMessageDto.to - the email address of the recipient
+   * @param createEmailMessageDto.template - the email template reference
+   * @param createEmailMessageDto.subject - the email message subject
+   * @param createEmailMessageDto.substitutions - object containing a dictionary of substitutions
+   * to be applied to the email message template
+   */
   createMessage(createEmailMessageDto: CreateEmailMessageDto): Promise<void>;
+
+  /**
+   * Deletes an email address previously persisted into the underlying provider.
```

Comment:
nitpick: I would phrase this more in a way that we trigger the deletion since deletion does not happen on our side.

Reply:
Changed in [e175e47](https://github.com/safe-global/safe-client-gateway/pull/871/commits/e175e4711a2ae2bb39fc88a25ac2188350fa8fdf) 🙂